changelogs/fragments/62587-module_utils-network-cloudengine.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - "The 'set-id' may change over the time althougth 'set-id' is the indentify of next rpc packet and also RPC xml attribute."
```

```suggestion
  - "Cloudengine modules: the 'set-id' may change over the time althougth 'set-id' is the indentity of next RPC packet and also RPC XML attribute."
```
You should always include what is affected by this change, otherwise almost all readers of the changelog will have no idea what this entry means.

---

Thanks, that is  a good idea.
I have update it.